build.gradle

```
@@ -107,6 +107,9 @@ tasks.withType(Test).configureEach {
         maxRetries = isCI ? 1 : 0
         maxFailures = 20
     }
+    predictiveSelection {
+        enabled = providers.gradleProperty("isPTSEnabled").map { it != 'false' } .orElse(false)
```

Minor formatting suggestion on my previously suggested minor formatting suggestion:

```suggestion
        enabled = providers.gradleProperty("isPTSEnabled").map { it != "false" }.orElse(false)
```